Makefile

```diff
@@ -47,6 +47,9 @@ OBJS_COMMON = $(BUILD_DIR)/ir.o $(BUILD_DIR)/ir_strtab.o $(BUILD_DIR)/ir_cfg.o \
 	$(BUILD_DIR)/ir_cpuinfo.o
 OBJS_IR = $(BUILD_DIR)/ir_main.o
 OBJS_IR_TEST = $(BUILD_DIR)/ir_test.o
+
+#FIXME(Tony): Will become terribly long when we have many many examples.
+#             Consider to use examples/Makefile of their own.
```

Comment:
Ditto. Guess this can be a separate improvement once more examples are filled. Similarly will be also to see on how to reorganize examples by topic, etc.

Thanks